### ISSUE TITLE
Version number in metadata file and database different

### ISSUE THREAD

**Issue by kripanshu** [TwoRavens/raven-metadata-service#128]
For custom_statistics:
When we update statistics on a given version let's say ( version: 1.1 ), it increases the number to +1 => version: 1.2 
` this is true only when 1.1 is the latest version, i.e there are no upper versions like ( 1.2,1.3 etc.)`
But if there is a scenario where a file has
`preprocess_id : 1, versions: 1.1,1.2,1.3. 
Now here we update statistics of the version 1.1 which should give version 1.4. 
But the metadata file is setting it's version 1.2 and database is setting it to 1.4`

**Comment by kripanshu** [comment]
I will make changes to custom_statistics branch.

**Comment by kripanshu** [comment]
In theory: 
- the person can always view their older versions
- make updates/deletes to change the latest back to an older version
So any updates always are against the latest.
**but**: don't change anything yet
---
Scenario:
- these versions exist:  1, 1.1, 1.2, 1.3
- the user has 2 editors open (`editor` being what UT is working on)
- one editor has version 1.3
- another browser window has version 1.1 loaded (somehow left it open by accident)
- *If the user submits an update from the 1.1 version, we should reject it* (edited)
In other words, the updates _should_ include the version to update--but this is only to check if it's the latest version
----
Shorter version - Same Scenario
- these versions exist:  1, 1.1, 1.2, 1.3
- update to 1.3 goes through
- update to 1, 1.1, or 1.2 fails b/c these aren't the latest version
---
This error should be rare but is definitely possible
---
by @raprasad
